droidlet/interpreter/tests/all_test_commands.py

```diff
@@ -986,22 +986,25 @@ def append_output(filt, output):
         "dialogue_type": "HUMAN_GIVE_COMMAND",
     },
     "dig a hole 2 times": {
-        "action_sequence" : [
+        "action_sequence": [
             {
                 "action_type": "DIG",
                 "remove_condition": {
                     "condition_type": "COMPARATOR",
                     "condition": {
                         "comparison_type": "EQUAL",
                         "input_left": {
-                            "value_extractor" : {
+                            "value_extractor": {
                                 "filters": {
                                     "output": {"attribute": "RUN_COUNT"},
```

Comment:
shouldn't this be "run_count" ? and not in capital, since we are  checking for "run_count" in keys of `CANONICALIZE_ATTRIBUTES` ?

Reply:
the code .lowers() the key: https://github.com/facebookresearch/droidlet/blob/updated_datasets/droidlet/interpreter/attribute_helper.py#L186